README.md

```diff
@@ -52,7 +52,8 @@ Result should be:
 ### Your solution here:
 4.  Fill in the `console.log()`?
 ```js
-console.log()
+console.log(firstWord+' '+secondWord+' '+thirdWord+' '+fourthWord);
```

Comment:
this should be written 

`console.log(firstWord + " " + secondWord + " " + thirdWord + " " + fourthWord);`